deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
Hi, to follow your comments, I undo'ed this change as well as the one in hack/testing/test-upgrade.sh.  Then, I reran the test using Rich's script oshift-aws-dev.sh.  But it fails in hack/testing/test-upgrade.sh.  To make this change, could there be any prerequisites?  Am I supposed to pick up some patch?  Thanks!